src/server/service/user.ts

```diff
@@ -22,6 +21,26 @@ export default class UserService implements IUserService {
 
   constructor(loggerService: ILoggerService) {
     this.loggerService = loggerService;
+    this.createAdmin();
```

Comment:
I don't think this procedure should be executed here.
This is the constructor for the `UserService` not for the Application itself.

The app is initialized in a separate process.

Reply:
`createAdmin` is a method for the `UserService` class, thus you are accessing a member of the class on its constructor